components/filter/filter.js

```diff
@@ -710,6 +715,17 @@ class Filter extends FocusMixin(LocalizeCoreElement(RtlMixin(LitElement))) {
 		this._activeDimensionKey = null;
 	}
 
+	_handleDimensionLoadMore(e) {
+		const dimensionKey = e.target.parentNode.id.slice(SET_DIMENSION_ID_PREFIX.length);
+
+		/** @ignore */
+		this.dispatchEvent(new CustomEvent('d2l-filter-dimension-load-more', {
```

Comment:
Would a consumer ever try to do this?
```html
<d2l-filter-dimension-set @d2l-filter-dimension-load-more=${...}>...</d2l-filter-dimension-set>
```
Are we sure the `<filter>` is the correct not to dispatch the event?

Reply:
Consumers would need to check for the key in the event. I replicated the behavior for the `d2l-filter-dimension-empty-state-action`, since rendering happens on filter it is also responsible for dispatching events.

Reply:
Yeah having the `filter` dispatch all the events is a side-effect of our choice to have filter render everything, which we recently met about and decided not to change, even with the limitations. I think we could _technically_ dispatch the event off the corresponding data-only dimension component, but like @GZolla mentioned the other dimension-specific events currently don't work this way.  I'm also not sure if the analyzer for the Daylight site would understand where to put the event in that case?